Arm/rover_arm_system.hpp

```diff
@@ -34,18 +34,26 @@ class RoverArmSystem : public sjsu::common::RoverSystem
   };
   struct MissionControlData : public RoverMissionControlData
   {
-    enum class Modes : char
+    enum class ArmModes : char
     {
       kHomeArm    = 'A',
       kHomeHand   = 'H',
       kConcurrent = 'C',
       kRotunda    = 'R',
       kShoulder   = 'S',
       kElbow      = 'E',
-      kWrist      = 'W'
-
+      kHand       = 'D'
+    };
+    ArmModes ArmMode = ArmModes::kConcurrent;
+    enum class HandModes : char
+    {
+      kPitch      = 'P',
+      kRoll       = 'R',
+      kClose      = 'F',
+      kOpen       = 'O',
+      kConcurrent = 'C'
     };
```

Comment:
Update [POR ](https://docs.google.com/presentation/d/17clPo-6g9CDImaTh3-akEh0AcbR9gbY4OoO5jGr4u0E/edit#slide=id.p)document (after slide ~76) with all these new modes. That way we can better understand each of these modes' purpose

Reply:
Done.